ui/modules/operations/operations.js

```diff
@@ -73,5 +73,18 @@ function createLoggerView(allLogLevels) {
       });
       dom.divLoggers.append(row);
     });
+    let newLoggerRow = document.createElement('div');
+    newLoggerRow.attachShadow({mode: 'open'});
+    newLoggerRow.shadowRoot.append(dom.templateLogger.content.cloneNode(true));
+    let inputLoggerElement = newLoggerRow.shadowRoot.getElementById('inputLogger');
+    inputLoggerElement.disabled = false;
+    inputLoggerElement.placeholder = 'Add new logger name and choose log level';
+    Array.from(newLoggerRow.shadowRoot.querySelectorAll('.btn-check')).forEach((btn) => {
+      btn.addEventListener('click', (event) => onUpdateLoggingClick(service, {
```

Comment:
"service" is then part of the anonymous function (?) Above I put "service" as an attribute to the button explicitly. I would make that consistently

Reply:
Sounds good. I did not manage to do it the same as you did. For some reasons, that did not work.